PED/SetPedMinGroundTimeForStungun.md

```diff
@@ -8,11 +8,84 @@ ns: PED
 void SET_PED_MIN_GROUND_TIME_FOR_STUNGUN(Ped ped, int ms);
 ```
 
+Overwrites the minimum time the ped will stay on the ground for after being stunned. Setting this while the ped is stunned will not alter the duration of the current stun (but will still effect future stunns).
+
+Unlike what this native's name suggests, it also effects the time for other "weapons", such as WEAPON_ELECTRIC_FENCE.
```

Comment:
Maybe let's change the native name then to better reflect the effect it has. Maybe something like SET_PED_MIN_GROUND_TIME_FOR_ STUN_EFFECT

Reply:
I think that SET_PED_MIN_GROUND_TIME_FOR_STUNGUN is the actual native name (since it does not have an underscore prefix denoting that it is a community-named native). Maybe @AvarianKnight has some input here to confirm it or correct me on this?

Reply:
We shouldn't change confirmed names.

Reply:
Sounds good and makes sense :)